### PR TITLE
setup project configuration

### PR DESCRIPTION
**Pull request by Andreat19**
Hi, This is my first tries on "Pull Request"

### CODE REVIEW

File: .github/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "cypress-training",
+  "version": "1.0.0",
+  "description": "This is a Workshop about Cypress",
+  "main": "[Por Defecto]",
```
Comment (holgiosalos) [inline .github/package.json]: Replace [Por Defecto] by index.js

File: .github/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "scripts": {
+    "test": "cypress open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "[Por Defecto]"
```
Comment (holgiosalos) [inline .github/package.json]: Replace `[Por Defecto]` by `git+https://github.com/Andreat19/-cypress-training.git`

**Review by holgiosalos — CHANGES_REQUESTED** [review]

File: .github/package.json (hunk 1 above)
Comment (Andreat19) [reply .github/package.json]: Done

File: .github/package.json (hunk 2 above)
Comment (Andreat19) [reply .github/package.json]: Done

File: .github/package.json (hunk 1 above)
Comment (Andreat19) [reply .github/package.json]: Done

File: .github/package.json (hunk 2 above)
Comment (Andreat19) [reply .github/package.json]: Done

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**